modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/display/context/BillingAddressCheckoutStepDisplayContext.java

```diff
@@ -43,7 +44,7 @@ public BillingAddressCheckoutStepDisplayContext(
 		CommerceOrder commerceOrder = getCommerceOrder();
 
 		return commerceAddressService.getBillingCommerceAddresses(
-			commerceOrder.getCompanyId(), CommerceAccount.class.getName(),
+			commerceOrder.getCompanyId(), AccountEntry.class.getName(),
```

Comment:
Is this AccountEntry change needed because of the fact Account are not Commerce entities anymore?
If this is the case, are there any other place where this change should be applied?